README.md

```diff
@@ -75,6 +75,15 @@ Extend real data from GitHub API with faked data based on extension IDL (you can
  * `--forward-headers`     Specify which headers should be forwarded to the proxied server
  * `--co`, `--cors-origin` CORS: Specify the origin for the Access-Control-Allow-Origin header
  * `-h`, `--help`          Show help
+ 
+When specifying the `[IDL file]` after the `--forward-headers` option you need to prefix it with `--` to clarify it's not another header. For example:
+```
+graphql-faker --forward-headers Authorition -- ./temp.faker.graphql
```

Comment:
You can't use `--forward-headers` without `--extend` so even though it's an example I think it's better to add `--extend` before `--forward-headers`

Reply:
Good point, added `--extend`.